PokeStar/PokeStar/Modules/RaidCommands.cs

```diff
@@ -189,7 +235,8 @@ public static async Task RaidReaction(IMessage message, SocketReaction reaction)
             {
                //help message - needs no update
                await player.SendMessageAsync(BuildRaidHelpMessage(message.Id));
-               await ((SocketUserMessage)message).RemoveReactionAsync(reaction.Emote, player);
+               await player.SendMessageAsync($"{message.Id}");
```

Comment:
give them the whole command to copy paste

Reply:
now gives .edit code